app/experimenter/nimbus-ui/src/components/PageResults/ExternalConfigAlert/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import ExternalConfigAlert from ".";
+import { MOCK_METADATA_EXTERNAL_CONFIG } from "../../../lib/visualization/mocks";
+
+describe("ExternalConfigAlert", () => {
+  it("renders as expected", () => {
+    render(
+      <ExternalConfigAlert
+        externalConfig={{
+          ...MOCK_METADATA_EXTERNAL_CONFIG,
+          reference_branch: "foo",
```

Comment:
I only changed this because I wanted to check for text context other than "control" in case we change "Baseline branch" text to use the word "control" later.